### PR TITLE
Ｍａｋｅ　ａｃｔｕａｌ　ｌｉｎｋ　ｉｎ　ＲＥＡＤＭＥ

### PR DESCRIPTION
**Pull request by richardanaya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ and the server must be restarted manually.
 * **ChaosBot has root access on its server.**  This means you are able to install
 packages and perform other privileged operations, provided you can initiate those
 changes through a pull request.
-* **It's domain name is chaosthebot.com,** but nothing is listening on
+* **It's domain name is [http://chaosthebot.com](http://chaosthebot.com),** but nothing is listening on
```
Comment (eighthjouster) [inline README.md:70]: It should be "Its" not "It's."

**Review by R01Ne — APPROVED** [review]

**Comment by amoffat** [comment]
i believe this PR is going to suffer from a similar issue described here https://github.com/chaosbot/chaos/pull/26 where your votes will be lost after merging/rebasing with upstream

**Comment by jabbink** [comment]
Shit title.

File: README.md (hunk 1 above)
Comment (shadowhand) [reply README.md:70]: `<http://chaosthebot.com>` would be shorter.

Comment (davidak) [reply README.md:70]: @eighthjouster that seams out of scope of this PR. you could make your own.

**Review by davidak — APPROVED** [review]

**Comment by Redmega** [comment]
@richardanaya Solve that merge conflict!

**Comment by jabbink** [comment]
already merged, could you please close this?

**Pull request closed without merging**